packages/nuxeo-web-ui-ftest/features/step_definitions/bulk_edit.js

```diff
@@ -0,0 +1,42 @@
+import { Then } from '@cucumber/cucumber';
+
+Then('I click the bulk edit button I see a dialog with editing options', function() {
+  this.ui.bulkEdit.editDocumentsButton.click();
+  this.ui.bulkEdit.dialog.waitForDisplayed();
+  this.ui.bulkEdit.bulkDefaultLayout.waitForDisplayed();
+  this.ui.bulkEdit.dialog.isVisible().should.be.true;
+  this.ui.bulkEdit.bulkDefaultLayout.isVisible().should.be.true;
+});
+
+Then('I can edit multiple properties:', function(table) {
```

Comment:
I think that this step (and any other step using the dialog and the bulk edit form, such as the `I can bulk edit all documents and {string} from {string} field` step), should be the ones waiting for the dialog to be visible, instead of assuming it will. This would make them a bit more reusable. This would also turn the `I click the bulk edit button I see a dialog with editing options` step into something simpler, like `I click the bulk edit button`.

Reply:
"I can edit multiple properties" is still a bit too generic. What about:
```suggestion
Then('I can bulk edit multiple properties:', function(table) {
```